ibm/mas_devops/roles/gencfg_sls/defaults/main.yml

```diff
@@ -5,4 +5,6 @@ mas_config_dir: "{{ lookup('env', 'MAS_CONFIG_DIR') }}"
 
 slscfg_url: "{{ lookup('env', 'SLSCFG_URL') }}"
 slscfg_registration_key: "{{ lookup('env', 'SLS_REGISTRATION_KEY') }}"
-slscfg_tls_crt: "{{ lookup('file', SLS_TLS_CERT_LOCAL_FILE) }}"
+
+sls_tls_crt_local_file_path: "{{ lookup('env', 'SLS_TLS_CERT_LOCAL_FILE_PATH') }}"
+slscfg_tls_crt: "{{ lookup('file', sls_tls_crt_local_file_path) }}"
```

Comment:
Can we remove this like we do in gencfg_uds and have it set in a fact at the start of the role. Otherwise we have two different mechanisms for similar things.